lib/gds_api/email_alert_api.rb

```diff
@@ -247,6 +247,24 @@ def send_subscription_verification_email(address:, frequency:, topic_id:)
     )
   end
 
+  # Unsubscribe all users for a subscription list
+  # optionally send a notification email explaining the reason
+  #
+  # @param [string]               slug                  SubscriptionList slug
+  # @param [string] (optional)    body                  Optional email body to send to alert users they are being unsubscribed
+  # @param [string] (optional)    sender_message_id     A UUID to prevent multiple emails for the same event
+  #                                                     generated as a v5 UUID comprising content_id + public_updated_at
+  #                                                     Email API requires this be provided if a body param is present.
+  def bulk_unsubscribe(slug:, body: nil, sender_message_id: nil)
```

Comment:
I think call this `subscriber_list_id`, since we use "slug" in so many places and for different things through the publishing system, and also it's what the `subscribe` method uses.